.github/workflows/built_and_test.yml

```diff
@@ -33,90 +24,72 @@ jobs:
           path: git_cache
           key: cache-wikibase-git-repo
 
-      - name: Update the git cache
-        run: bash update_cache.sh
-
       - name: Build Tarball
-        run: bash build_scripts/build_wikibase.sh
+        run: bash build_github.sh wikibase
 
       - name: Archive tar production artifacts
         uses: actions/upload-artifact@v2
         with:
           name: Wikibase Tarball
-          path: ${{env.TARBALL_PATH}}
-
-      - name: Build Docker image
-        run: bash build_scripts/build_wikibase_docker.sh ${{ env.WIKIBASE_IMAGE_NAME }}
+          if-no-files-found: error
+          path: artifacts/Wikibase.tar.gz
 
       - name: Archive docker production artifacts
         uses: actions/upload-artifact@v2
         with:
           name: Wikibase Docker Image
-          path: wikibase.docker.tar.gz
+          if-no-files-found: error
+          path: artifacts/wikibase.docker.tar.gz
 
   build_queryservice:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-
+      - uses: c-py/action-dotenv-to-setenv@v2
       - uses: satackey/action-docker-layer-caching@v0.0.8
         continue-on-error: true
         with:
           key: queryservice-docker-cache-${{env.DOCKER_CACHE_VERSION}}-{hash}
           restore-keys: queryservice-docker-cache-${{env.DOCKER_CACHE_VERSION}}-
 
-      - name: Download QueryService Tarball
+      - name: Build QueryService Image 
         id: download_step
-        run: bash build_scripts/build_queryservice.sh
-
-      - name: Archive tar production artifacts
```

Comment:
Since we are not altering the tarball it makes little sense to have this step?

Reply:
I guess it can go for now; we probably will be altering it when we add our changelog though?